prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/ChannelCommandTest.java

```diff
@@ -191,7 +191,33 @@ public void testList() {
         int exitCode = commandLine.execute(CliConstants.Commands.CHANNEL, CliConstants.Commands.LIST,
                 CliConstants.DIR, dir.toString());
         Assert.assertEquals(ReturnCodes.SUCCESS, exitCode);
-        Assert.assertEquals(3, getStandardOutput().lines().filter(l->l.contains("manifest")).count());
+        Assert.assertEquals(3, getStandardOutput().lines()
+                .filter(line -> line.matches(".*\\S+ \\S+:\\S+:\\S+"))  // Check if the line contains a name and manifest details
+                .count());
+    }
+
+    @Test
+    public void testFullList() {
+        int exitCode = commandLine.execute(CliConstants.Commands.CHANNEL, CliConstants.Commands.LIST, CliConstants.FULL,
+                CliConstants.DIR, dir.toString());
+        Assert.assertEquals(ReturnCodes.SUCCESS, exitCode);
+        String output = getStandardOutput();
+        Assert.assertTrue("Expected 'schemaVersion' in the output",
```

Comment:
The test doesn't actually verify that the correct values are printed.  You should be able to get the output, convert it back to Channel objects and compare with the input. 
Or simply compare the output with serialized input.